libs/eo/certificates/src/lib/eo-certificates.store.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { HttpErrorResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import {
+  EoCertificate,
+  EoCertificatesService,
+} from './eo-certificates.service';
+
+interface EoCertificatesState {
+  loadingDone: boolean;
```

Comment:
Consider using a naming convention for booleans indication a boolean value.
Eg. prefix with "is" or "has".
Examples: "hasLoaded" or "isLoading"

Reply:
Fixed :)